### PR TITLE
#1015 created test for issue

### PR DESCRIPTION
**Pull request by krzyk**
#1015 
- created test for isse with manual assignement of PRs
- added todo for implementation

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#1150](https://github.com/zerocracy/farm/issues/1150) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#1150](https://github.com/zerocracy/farm/issues/1150) is assigned to @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda), here is [why](https://www.0crat.com/footprint/C3NDPUA8L/1642163447); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @g4s8[/z](https://www.0crat.com/u/g4s8) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

**Comment by krzyk** [comment]
@carlosmiranda ping

File: src/test/resources/com/zerocracy/bundles/assigns_performer_pr/_after.groovy [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+import com.zerocracy.pm.in.Orders
+
+def exec(Project project, XML xml) {
+  def job = 'gh:test/test#1'
+  assert new Orders(project).bootstrap().performer(job) == 'krzyk'
+  // @todo #1015:30min Fix the problem when PR job is assigned manually it
+  //  has role of DEV and 30 points award. After fixing it uncomment following
+  //  two lines and the test should pass.
+  //assert new Wbs(project).bootstrap().role(job) == 'REV'
+  //assert new Boosts(project).bootstrap().factor(job) * 15 == 15
```
Comment (carlosmiranda) [inline src/test/resources/com/zerocracy/bundles/assigns_performer_pr/_after.groovy]: @krzyk we should just check boost factor here. Maybe in the future we will change the base value of 1x boost, and we are hard coding it here to mean 15 minutes.

**Comment by carlosmiranda** [comment]
@krzyk one comment. Apologies for the delay, had some urgent "IRL" issues to take care of. :)

**Comment by krzyk** [comment]
@carlosmiranda Yeah, I figured from the lack of github activity :)
Updated the code, please check.

**Comment by 0crat** [comment]
@ypshenychka[/z](https://www.0crat.com/u/ypshenychka) please review this job completed by @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
The job [#1150](https://github.com/zerocracy/farm/issues/1150) is now out of scope

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @g4s8[/z](https://www.0crat.com/u/g4s8)

**Comment by ypshenychka** [comment]
@carlosmiranda According to our [QA Rules](http://www.zerocracy.com/policy.html#42):
>The code reviewer found at least three problems in the code.
Comments were mostly about design problems, not cosmetic issues.

Only one issue was found during code review. 
Please confirm that you'll try to find at least three major problems while future reviews.

**Comment by carlosmiranda** [comment]
@ypshenychka confirmed

**Comment by ypshenychka** [comment]
@carlosmiranda thank you

**Comment by ypshenychka** [comment]
@0crat quality acceptable

**Comment by 0crat** [comment]
Order was finished, quality is "acceptable": +15 point(s) just awarded to @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda)

**Comment by 0crat** [comment]
Quality review completed: +8 point(s) just awarded to @ypshenychka[/z](https://www.0crat.com/u/ypshenychka)

**Pull request merged**